.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: elixir
+elixir: '1.6.5'
+otp_release:
+  - '20.0'
+  - '20.1'
+  - '20.2'
+  - '20.3'
```

Comment:
I think we'll also need to specify some build steps as well. I think `mix compile`, `mix credo` and `mix dialyzer` could be good for now.

Reply:
added them in the yml file